Codeinwp/otter-blocks
# Posts block with Automatic and FIFU plugins

Closes #1425.


### Summary

The error occurred because
```javascript
const image = select( 'core' ).getMedia( id, { context: 'view' });
```
does not return the expected object
This PR adds a new check to make sure we don't try to access `image.media_details.sizes` when not possible.
### Screenshots 

----

### Test instructions

- Activate FIFU and Automatic plugins
- From Automatic add a New Campaign of type Amazon with the following settings checked:

<img width="478" alt="Screenshot 2023-01-26 at 14 13 19" src="https://user-images.githubusercontent.com/39873395/214832545-14401191-1876-4773-8874-d892a786d1c3.png">

- On a page add a Posts block from Otter
-  Run the campaign created earlier
- After going again to the post block and clicking it should not throw any error



---- 

### Checklist before the final review
